definitions/benefit/sheets/ComplexTypes/ComplexTypes.json

```diff
@@ -544,5 +544,10 @@
   {"LiveFrom": "13/01/2025", "ID": "hearingOutcomeDetails", "ListElementCode": "completedHearings",  "FieldType": "DynamicList", "ElementLabel": "Hearings", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/10/2024", "ID": "hearingOutcomeValue", "ListElementCode": "hearingOutcomeId", "FieldType": "FixedList", "FieldTypeParameter": "FL_caseOutcome", "ElementLabel": "Hearing Outcome", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/10/2024", "ID": "hearingOutcomeValue", "ListElementCode": "didPoAttendHearing", "FieldType": "YesOrNo", "ElementLabel": "Did PO Attend?", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "18/10/2024", "ID": "hearingOutcomeValue", "ListElementCode": "completedHearings", "FieldType": "DynamicList", "ElementLabel": "Hearing", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "18/10/2024", "ID": "hearingOutcomeValue", "ListElementCode": "completedHearings", "FieldType": "DynamicList", "ElementLabel": "Hearing", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2025", "ID": "ftaRequest", "ListElementCode": "requestDueDate", "FieldType": "DateTime", "ElementLabel": "Request Due Date Time", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2025", "ID": "ftaRequest", "ListElementCode": "requestDateTime", "FieldType": "DateTime", "ElementLabel": "Request Start Date and Time", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2025", "ID": "ftaRequest", "ListElementCode": "requestTopic", "FieldType": "FixedList", "FieldTypeParameter": "FL_ftaRequestTopic", "ElementLabel": "Request Topic", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2025", "ID": "ftaRequest", "ListElementCode": "requestText", "FieldType": "TextArea", "ElementLabel": "Request Text", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2025", "ID": "ftaRequest", "ListElementCode": "requestUserName", "FieldType": "Text", "ElementLabel": "Request User Name", "SecurityClassification": "PUBLIC"}
```

Comment:
change line 552 "ElementLabel": "Request User Name" to "ElementLabel": "Request Submitted By"

Reply:
done